packages/tiptap/src/components/editor/editor.tsx

```diff
@@ -0,0 +1,82 @@
+import Bold from '@tiptap/extension-bold';
+import Document from '@tiptap/extension-document';
+import HorizontalRule from '@tiptap/extension-horizontal-rule';
+import Italic from '@tiptap/extension-italic';
+import ListItem from '@tiptap/extension-list-item';
+import ListKeyMap from '@tiptap/extension-list-keymap';
+import Paragraph from '@tiptap/extension-paragraph';
+import Strike from '@tiptap/extension-strike';
+import Text from '@tiptap/extension-text';
+import { EditorContent, useEditor } from '@tiptap/react';
+import ImageResize from 'tiptap-extension-resize-image';
+
+import {
+  CustomBulletListConfigure,
+  CustomBulletListExtend,
+  CustomHeading,
+  CustomImageConfigure,
+  CustomLinkConfigure,
+  CustomLinkExtend,
+  CustomOrderedListConfigure,
+  CustomTextAlignConfigure,
+  TabIndentExtension,
+} from '../../utils/custom-extensions.ts';
+
+import MenuBar from '../../components/menu-bar/menu-bar.tsx';
+
+interface EditorProps {
+  editorContent: string;
```

Comment:
```suggestion
  content: string;
```

`editorContent`에서 editor 프리픽스를 넣은 이유가 있나요?

Reply:
추후에 에디터를 게시글 작성 페이지에서 사용할 때 겹칠 수도 있을 듯 하여 구분감을 주기 위해 넣었습니다!